client/src/apiClient.js

```diff
@@ -1,29 +1,32 @@
 const baseUrl = 'http://localhost:8080';
 
 class apiClient {
-
-  addMessage = async (data) => {
-      const response  = await fetch(`${baseUrl}/api/message/add`, {
-          method: 'POST',
-          headers: {"content-type": "application/json"},
-          mode: 'cors',
-          body: JSON.stringify(data)
-      })
-      return response.json();
-  };
-
-
-  getMessages = async (data) => {
-    const response  = await fetch(`${baseUrl}/api`, { //route is specified in  server/api/index
-        method: 'GET',
-        headers: {"Accept": "application/json"},
-        mode: 'cors',
-    })
-    return response.json();
-  };
-};
-
-
-
+    addMessage = async data => {
+        const response = await fetch(`${baseUrl}/api/message/add`, {
+            method: 'POST',
+            headers: { 'content-type': 'application/json' },
+            mode: 'cors',
+            body: JSON.stringify(data),
+        });
+        return response.json();
+    };
+
+    getMessages = async data => {
+        const response = await fetch(`${baseUrl}/api`, {
+            //route is specified in  server/api/index
+            method: 'GET',
+            headers: { Accept: 'application/json' },
+            mode: 'cors',
+        });
+        return response.json();
+    };
+
+    deleteMessage = async id => {
+        const response = await fetch(`${baseUrl}/api/message/delete/${id}`, {
```

Comment:
We don't need to have the `delete` word in the route, it' already stated with the http method